fetch.bs

```diff
@@ -4796,7 +4804,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
  <li><p>Let <var>response</var> be null.
 
- <li><p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a>, given
+ <li><p>Let <var>httpCache</var> be the result of <a>determining the HTTP state partition</a>, given
```

Comment:
Let <var>httpCache</var> be the HTTP cache associated with the result of <a>determining the HTTP cache partition</a>

Reply:
Thanks! Meant to put the line below this above it, but even that doesn't really handle the null case. Fixed.